README.md

```diff
@@ -1,2 +1,7 @@
 # weather-fun
 An example project for the revision control training course
+
+
+The aim of this project is to get a hands-on experience with classifying weather data to accurately represent the weather conditions in Cambridge. This is going to be used the find the weather condition given dome parameters (i.e. sunshine, rainfall and wind).
```

Comment:
I think there is a typo here - should be "some" not "dome"

Reply:
Thanks for letting me know! I fixed it now!